bundles/org.openhab.binding.http/README.md

```diff
@@ -0,0 +1,118 @@
+# HTTP Binding
+
+This binding allows using HTTP to bring external data into openHAB or execute HTTP requests on commands.  
+
+## Supported Things
+
+Only one thing named `url` is available.
+It can be extended with different channels.
+
+## Thing Configuration
+
+| parameter  | optional | default | description |
+|------------|----------|---------|-------------|
+| `baseURL`  | no       |    -    | The base URL for this thing. Can be extended in channel-configuration. |
+| `refresh`  |          |   30    | Time in seconds between two refresh calls for the channels of this thing. |
+| `timeout`  |          | 3000    | Timeout for HTTP requests in ms. |
+| `username` | yes      |         | Username for authentication (advanced parameter). |
+| `password` | yes      |         | Password for authentication (advanced parameter). |
+| `authMode` |          | BASIC   | Authentication mode, `BASIC` or `DIGEST` (advanced parameter). |
```

Comment:
You should fill in the `optional` column for every parameter here, even if it would be implied by the presence of a default value.